src/wait.ts

```diff
@@ -29,6 +30,7 @@ export class Waiter implements Wait {
       (secondsSoFar || 0) >= this.input.continueAfterSeconds
     ) {
       this.info(`🤙Exceeded wait seconds. Continuing...`);
+      exportVariable("CI_SKIP", "true");
```

Comment:
This is a cool idea. I've seen the more common case be that actions set an output the can be consumed by other steps. Could we use an output variable instead?

Reply:
Sounds good. This is done!